src-tauri/src/command/mod.rs

```diff
@@ -51,16 +49,13 @@ async fn create_link(
 /// Remove a `.link` from directory
 async fn delete_link(name: String) -> Result<()> {
     let path = ARK_SHELF_WORKING_DIR.get().unwrap().join(&name);
+    let content = fs::read_to_string(&path)?;
+    let id = arklib::id::ResourceId::compute_bytes(content.as_bytes())
```

Comment:
Yep :+1: Later we'll integrate with the index from `arklib` so we won't recompute the ids all the time.

Reply:
the name is currently: domain-id. That's why I used to split with the - to get the id without rereading the file before

Reply:
Yeah, I understand. Still, it's less direct than counting id from the content itself, and also URLs are small enough to compute their id quickly.